improved lcluster init

F1r3Pr1nc3 opened the pull request:
Optimizing for the L-Cluster initializing (which leads to a lag on creation). Every L-Cluster system is saved in a temporary global target, but only AFTER the full creation of the L-Cluster, where after that, EVERY system is scanned again for EVERY galaxy system (which is multiplication). I changed this to the common way on saving this systems immediately in the init_effect (so without needless 9 galaxy system scans). I feel a noticeable improvement now.

F1r3Pr1nc3 committed:
fitting is_same_empire

F1r3Pr1nc3 committed:
improved lcluster init

F1r3Pr1nc3 committed:
improved lcluster init for graygoo

remove needless random_system

Ariphaos commented:
Heya, two requests:

1) Could you get rid of the whitespace changes?

2) Add some lines to the changelog explaining your fixes (and tag your name for fame and blame : )

F1r3Pr1nc3 committed:
fix Mass Extinction Event Chain

https://forum.paradoxplaza.com/forum/threads/stellaris-mass-extinction-through-the-ages-bugged.1463688

F1r3Pr1nc3 commented:
> 1. Could you get rid of the wh

Hm sorry, this is an automated thing from CWtools, but my Git has an option to ignore whitespaces!? :-P

F1r3Pr1nc3 committed:
Merge pull request #1 from Ariphaos/master

Pushing recent fixes to Github
